ext/nnstreamer/tensor_filter/tensor_filter_cpp.h

```diff
@@ -49,20 +49,22 @@ class tensor_filter_cpp {
 
   public:
     tensor_filter_cpp(const char *modelName); /**< modelName is the model property of tensor_filter, which could be the path to the model file (requires proper extension name) or the registered model name at runtime. */
-    ~tensor_filter_cpp();
+    virtual ~tensor_filter_cpp();
 
     /** C++ plugin writers need to fill {getInput/Output} inclusive-or {setInput} */
     virtual int getInputDim(GstTensorsInfo *info) = 0;
     virtual int getOutputDim(GstTensorsInfo *info) = 0;
 
-    virtual int setIutputDim(const GstTensorsInfo *in, GstTensorsInfo *out) = 0;
+    virtual int setInputDim(const GstTensorsInfo *in, GstTensorsInfo *out) = 0;
 
     bool allocate_before_invoke; /**< TRUE, if you want nnstreamer to preallocate output buffers before calling invoke */
```

Comment:
[Discussion] about changing this flag as private variable, and updating constructor to set this.

Reply:
I'll set it protected because its derived class (cpp custom filter developer's class) should be able to set it.

Reply:
or maybe.. make it const

Reply:
Resolved by adding a pure-virtual get function